porch/config/deploy/5-rbac.yaml

```diff
@@ -30,3 +30,7 @@ rules:
 - apiGroups: ["flowcontrol.apiserver.k8s.io"]
   resources: ["flowschemas", "prioritylevelconfigurations"]
   verbs: ["get", "watch", "list"]
+# Needed to launch / read executor pods
```

Comment:
I'm not sure if it is possible to restrict to the `porch-functions-system` namespace. Do you happen to know?

Reply:
Good call - it is.  We would just create a RoleBinding rather than a ClusterRoleBinding.  A RoleBinding can bind either to a ClusterRole or a Role, but then only binds in that Namespace.